open-company/open-company-web
# UX desktop: send read with comments

Card: https://trello.com/c/gZGifMOU

To test:
- user A open AP, user B keep it close
- with user A add a post with only one line
- with user B open AP
- [x] user A sees read? Good
- close AP with user B
- with user A add a post with only one line
- user A comment on the post
- with user B open AP
- [x] user A sees read? Good
- close AP with user B
- with user A add a long post (at least 5 lines)
- with user B open AP
- [x] user A doesn't see read? Good
- user B expand post
- [x] user A sees read? Good
